src/Strategy/AlpineLinux/Parser.hs

```diff
@@ -0,0 +1,128 @@
+-- | Parsing functions for APK files.
+-- APK is the package manager for Alpine Linux (https://alpinelinux.org/).
+--
+-- The database itself is just a flat file with entries in text.  These are
+-- specified in a key-value format with obscure names.  See
+-- https://wiki.alpinelinux.org/wiki/Apk_spec#Syntax.
```

Comment:
Can we add a trivial inline example to these docs?
Something like this, but with explanations of what we do with the first set of kv pairs?  Which ones we read, which ones we discard, if there are relevant ones we don't use yet, etc.

The idea is to prevent what you ran into with having to read an obscure spec that just links to code, or making the reader read THIS module's code.
```
C: whatever
P: sompackage
A: someattr
f: foobar
V: 1.2.3

<another package>

```